src/main/scala/atom/config-schema/SettingsGroup.scala

```diff
@@ -3,16 +3,15 @@ package laughedelic.atom.config
 import scala.scalajs.js
 
 class SettingsGroup[S <: ConfigSchema](
+  schema: S,
   val title: js.UndefOr[String] = js.undefined,
-  val description: js.UndefOr[String] = js.undefined,
```

Comment:
Atom [API docs](https://atom.io/docs/api/v1.24.0/Config#title-and-description) say:

> For a group of config settings the humanized key or the title and the description are used for the group headline.

But in practice group's description is not rendered. So I removed it here. Probably it's just a bug in Atom, idk.